packages/IMAPClient-Core.package/ICEndPoint.class/class/setFlagSeen..st

```diff
@@ -0,0 +1,4 @@
+commands
+setFlagSeen: messageId
+
+	^ {self protocolCommandStart . 'STORE' . messageId . '+FLAGS \Seen'} joinSeparatedBy: self protocolBlank
```

Comment:
In `resetFlagSeen`, `setFlagFlagged` and `setFlagSeen` is a lot of code duplication. Couldnt we refactor it into a method, which gets two string (`\Flagged` or `\Seen` and `-Flags` or `+Flags`) and inserts them into the command? Everything else is exactly equal.

Reply:
I did this on purpose, since it is a class-method and command for the imap-server. Also to be it consistent with similar methods that were implemented by other groups.